MISC/ClearAreaOfCops.md

```diff
@@ -5,17 +5,22 @@ ns: MISC
 
 ```c
 // 0x04F8FC8FCF58F88D 0x95C53824
-void CLEAR_AREA_OF_COPS(float x, float y, float z, float radius, int flags);
+void CLEAR_AREA_OF_COPS(float x, float y, float z, float radius, cs_type(int) BOOL createNetEvent);
 ```
 
-```
-flags appears to always be 0
+Clears an area of cops at the given coordinates and radius.
+`createNetEvent` specifies whether a `CClearAreaEvent` event of should be created for online use.
```

Comment:
Is this line necessary considering its already defined in the parameters? (same goes for the other natives)

Reply:
I just corrected it, thanks!